compiler/rustc_codegen_ssa/src/back/write.rs

```diff
@@ -913,7 +913,9 @@ fn execute_copy_from_cache_work_item<B: ExtraBackendMethods>(
 
     let object = load_from_incr_comp_dir(
         cgcx.output_filenames.temp_path(OutputType::Object, Some(&module.name)),
-        module.source.saved_files.get("o").expect("no saved object file in work product"),
+        module.source.saved_files.get("o").unwrap_or_else(|| {
+            cgcx.create_dcx().emit_fatal(errors::NoSavedObjectFile { cgu_name: &module.name })
```

Comment:
Is "CGU" the rustc term for what LLVM calls a "module"? I'm trying to understand the `cgu_name`/`module.name` inconsistency here.

Reply:
Yes, I think that's what's going on here. I decided to call this a CGU name based on how `CachedModuleCodegen` are created: https://github.com/rust-lang/rust/blob/b5c46dc5426038a49c95398bce30eeb20ec421e2/compiler/rustc_codegen_ssa/src/base.rs#L770-L773

Reply:
👍